### ISSUE TITLE
[QUESTION] librespot addon available on a general installation of Kodi

### ISSUE THREAD

**Issue by e-ru** [LibreELEC/LibreELEC.tv#5149]
Hello LibreELEC community

I know this is a completely unrelated question. But, besides of LibreELEC, I'm also using Kodi on a more general machine and over the last couple of days I was trying to get the librespot addon running - without success.

Here is some background

For now Spotifyd is the Spotify connect client on the mentioned machine. This works well, but it runs in parallel to Kodi, so no volume control via Kodi.
My idea now was to simply get the librespot addon from an x86_64 build of LibreELEC and install this zip on my machine on a general Kodi installation, but it's not that simple.
With enabled discovery the addon panics with a DNSError of the librespot rust module.
With disabled discovery I am able to connect to the addon via the Spotify App, but as soon as a song starts playing there is another rust related panic.

To have this addon also available on a general installation of Kodi would be great, but I definitely need some help with this.

Maybe @awiouy sees this issue and has lust and time to have a look into it together with me.

Sorry for this unrelated question and thank your work on LibreELEC.
Erik

**Comment by heitbaum** [comment]
@e-ru please have a look at _Originally posted by @heitbaum in https://github.com/LibreELEC/LibreELEC.tv/issues/5038#issuecomment-807977909_ there is a test x86 binary there. Looks like the issue with vorbis has been resolved as tested by the RPI4 users. Hope that zip helps you in your quest :-)

I will be deleting that GitHub tree (and zips) once the PR is merged into master.

#5038
#5166 

pill close this question based on the above. Please reach out if there is anything else.
